server/webhook_parser.go

```diff
@@ -405,6 +416,34 @@ func parseWebhookUpdatedField(jwh *JiraWebhook, eventType string, field, fieldID
 	return wh
 }
 
+func parseWebhookUpdatedEpicLink(jwh *JiraWebhook, field, fieldID, from, fromID, to, toID string, p *Plugin, instanceID types.ID) *webhook {
```

Comment:
I think it makes sense to have the larger dependencies listed first

```suggestion
func parseWebhookUpdatedEpicLink(jwh *JiraWebhook, p *Plugin, instanceID types.ID, field, fieldID, from, fromID, to, toID string) *webhook {
```